nengo_viz/swi.py

```diff
@@ -220,7 +220,7 @@ def handle_ws_request(self, args, db):
         key = key.encode('ascii')
         resp_data = (HSHAKE_RESP %
                      base64.b64encode(hashlib.sha1(key).digest()).decode('ascii'))
-        client.socket.send(resp_data.encode('ascii'))
+        client.socket.sendall(resp_data.encode('ascii'))
```

Comment:
socket is still in nonblocking mode here, so we can use `socket.sendall` instead of `ClientSocket._sendall`